src/MyCocktails/MyCocktails.tsx

```diff
@@ -1,9 +1,23 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
+import { getAllCocktails } from '../apiCalls';
+// import './MyCocktails.scss';
+
 
 interface MyCocktailsProps {
 }
 
 const MyCocktails: React.SFC<MyCocktailsProps> = () => {
+	const [allCocktails, setAllCocktails] = useState({})
+
```

Comment:
Notes for adding array of recipes, not functional at all, more of a mental roadmap.